Stop running CI tests against master branch of PTVSD

DonJayamanne opened the pull request:
For #3414

<!--
  If an item below does not apply to you, then go ahead and check it off as "done" and strikethrough the text, e.g.:
    - [x] ~Has unit tests & system/integration tests~
-->
- [x] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR)
- [x] Title summarizes what is changing
- [x] Has a [news entry](https://github.com/Microsoft/vscode-python/tree/master/news) file (remember to thank yourself!)
- [no] Unit tests & system/integration tests are added/updated
- [no] [Test plan](https://github.com/Microsoft/vscode-python/blob/master/.github/test_plan.md) is updated as appropriate
- [no] [`package-lock.json`](https://github.com/Microsoft/vscode-python/blob/master/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed)

brettcannon commented on .travis.yml:
```diff
@@ -68,7 +59,7 @@ install:
   - npx gulp hygiene-modified
   - python -m pip install --upgrade -r requirements.txt
-  - python -m pip install -t ./pythonFiles/experimental/ptvsd git+https://github.com/Microsoft/ptvsd/
+  - python -m pip install -t ./pythonFiles/experimental/ptvsd ptvsd --no-cache-dir;
```
```suggestion
  - python -m pip install -t ./pythonFiles/experimental/ptvsd --no-cache-dir --implementation py --no-deps ptvsd
```

brettcannon commented on news/3 Code Health/3414.md:
```diff
@@ -0,0 +1 @@
+Stop running CI tests against master branch of PTVSD.
```
```suggestion
Stop running CI tests against the `master` branch of ptvsd.
```

d3r3kk commented:
@DonJayamanne does this mean we should shut down the Azure DevOps pipeline that does this as well?

DonJayamanne commented:
Already done (I modified the ado and when reviewing your yaml PR I did mention this and we fixed it there to) if not please do so. It
To be clear it should only run against release version of ptvsd.
